src/supervisor/watchers/handlers/index.ts

```diff
@@ -11,32 +11,59 @@ import {
 
 import { logger } from '../../../common/logger';
 import { WorkloadKind } from '../../types';
-import { podWatchHandler, podDeletedHandler, paginatedPodList } from './pod';
+import {
+  podWatchHandler,
+  podDeletedHandler,
+  paginatedNamespacedPodList,
+  paginatedClusterPodList,
+} from './pod';
 import {
   cronJobWatchHandler,
-  paginatedCronJobList,
-  paginatedCronJobV1Beta1List,
+  paginatedClusterCronJobList,
+  paginatedClusterCronJobV1Beta1List,
+  paginatedNamespacedCronJobList,
+  paginatedNamespacedCronJobV1Beta1List,
 } from './cron-job';
-import { daemonSetWatchHandler, paginatedDaemonSetList } from './daemon-set';
-import { deploymentWatchHandler, paginatedDeploymentList } from './deployment';
-import { jobWatchHandler, paginatedJobList } from './job';
-import { paginatedReplicaSetList, replicaSetWatchHandler } from './replica-set';
 import {
-  paginatedReplicationControllerList,
+  daemonSetWatchHandler,
+  paginatedClusterDaemonSetList,
+  paginatedNamespacedDaemonSetList,
+} from './daemon-set';
```

Comment:
maybe the exports could be changed to import just 1 thing e.g. `daemonSet` and then could call the methods from that e.g. `demonSet.watchHandler`, `demonSet.paginatedSetList` and `demonSet.paginatedNamespacedSetList` ... also could drop the paginated all together and just have comment in the file?